manifests/m/Microsoft/VisualStudio/2010/ToolsOfficeRuntime/60828.00/Microsoft.VisualStudio.2010.ToolsOfficeRuntime.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with Komac v1.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Microsoft.VisualStudio.2010.ToolsOfficeRuntime
+PackageVersion: '60828.00'

```

Comment:
```suggestion
PackageVersion: '10.0.60833'
```

Reply:
Control Panel shows **10.0.60828** for both x86 and x64 architectures, so I am using this for PackageVersion.

'PS> Get-WmiObject Win32_Product' returns **10.0.60833**, which I am using on each architecture DisplayVersion:

**x64:**
IdentifyingNumber : {47C2CCDB-7A04-3797-992B-A84D3E90258F}
Name              : Microsoft Visual Studio 2010 Tools for Office Runtime (x64)
Vendor            : Microsoft Corporation
**Version           : 10.0.60833**
Caption           : Microsoft Visual Studio 2010 Tools for Office Runtime (x64)


**x86:**
IdentifyingNumber : {39ABE1E5-A84D-3FC5-8311-776474862F50}
Name              : Microsoft Visual Studio 2010 Tools for Office Runtime (x86)
Vendor            : Microsoft Corporation
**Version           : 10.0.60833**
Caption           : Microsoft Visual Studio 2010 Tools for Office Runtime (x86)